roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/configure-worker-node.yml

```diff
@@ -14,6 +9,22 @@
 
 # How to wait for node to be scheduleable? (NodeSchedulable)
 - name: Wait for all k8s nodes to be ready
-  ansible.builtin.command: oc wait --for=condition=Ready nodes --all --timeout=3600s
-  register: __sap_hypervisor_node_preconfigure_register_nodes_ready
-  changed_when: __sap_hypervisor_node_preconfigure_register_nodes_ready.rc != 0
+  kubernetes.core.k8s_info:
+    host: "{{ sap_hypervisor_node_preconfigure_ocp_endpoint }}"
+    ca_cert: "{{ sap_hypervisor_node_preconfigure_ocp_ca_cert }}"
+    api_key: "{{ __sap_hypervisor_node_preconfigure_register_openshift_auth_results.openshift_auth.api_key }}"
+    kubeconfig: "{{ sap_hypervisor_node_preconfigure_ocp_kubeconfig }}"
```

Comment:
Isn't setting `host` + `ca_cert` + `api_key` and `kubeconfig` redundant?

Reply:
I thought the same, but the k8s module is unhappy if either of them is not set. This might be a bug there.

Reply:
Setting all in enviroment though is possible and is kind of a workaround for that, too.